coverage/.last_run.json

```diff
@@ -1,5 +1,5 @@
 {
   "result": {
-    "covered_percent": 83.44
+    "covered_percent": 27.78
```

Comment:
We really need to nail down what's up with your test environment @mboneil10.

Reply:
Yes.